### PR TITLE
NIT-710 alfresco alert when 400 errors reach high levels

### PR DESCRIPTION
**Pull request by pete-j-g**

### CODE REVIEW

File: monitoring/alarms.tf [hunk 1]
```diff
@@ -159,3 +159,40 @@ resource "aws_cloudwatch_metric_alarm" "free_storage_space_warning" {
     DBInstanceIdentifier = local.db_instance_id
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "content_4xx_anomaly_detection" {
+  alarm_name          = "${local.application}_content_tg_4xx-error-anomaly"
+  alarm_description   = "Database free storage space less than 10% of total storage space. Please contact ${local.support_team}"
```
Comment (kylehodgetts) [inline monitoring/alarms.tf]: I think this needs to be updated to refer to the ELB

File: monitoring/alarms.tf [hunk 2]
```diff
@@ -159,3 +159,40 @@ resource "aws_cloudwatch_metric_alarm" "free_storage_space_warning" {
...
+  alarm_actions       = [aws_sns_topic.alarm_notification.arn]
+  ok_actions          = [aws_sns_topic.alarm_notification.arn]
+  comparison_operator = "GreaterThanUpperThreshold"
+  threshold_metric_id = "ad1"
+  evaluation_periods  = 1
+  treat_missing_data  = "ignore"
+  datapoints_to_alarm = local.datapoints_to_alarm
+
+  metric_query {
+    id          = "m1"
```
Comment (kylehodgetts) [inline monitoring/alarms.tf:176]: m1?

Comment (kylehodgetts) [reply monitoring/alarms.tf:176]: metric1

**Review by kylehodgetts — APPROVED** [review]

File: monitoring/alarms.tf (hunk 1 above)
Comment (pete-j-g) [reply monitoring/alarms.tf]: bad copypasta. fixed

**Pull request merged**